workers/unity/Packages/com.improbable.gdk.core/Logging/Logger.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class Logger : ILogger
+    {
+        private readonly ILogDispatcher dispatcher;
```

Comment:
We are we adding our a wrapper around our own logger and using another similarly named interface?

Reply:
I added this as an attempt to get rid of the pattern where every log would have to pass in the class it was logging for. I'm open for suggested alternatives.

Reply:
Why couldn't the wrapper also implement ILogDispatcher?

Reply:
Maybe the `ILogger` interface was a bit much, I could just make it a `Logger` directly? Its simply meant to be a tool to de-duplicate the class field passing.